includes/components/class-attribute.php

```diff
@@ -398,6 +402,7 @@ public function register_attributes() {
 						'moderated'      => (bool) $attribute_object->hp_moderated,
 						'indexable'      => (bool) $attribute_object->hp_indexable,
 						'searchable'     => (bool) $attribute_object->hp_searchable,
+						'registrable'    => (bool) $attribute_object->hp_registrable,
```

Comment:
There was probably a confusion because of the "or maybe a separate flag is required for this?" question, please rewrite the code to implement this without a flag, a condition for fields to appear in the registration form is: the field is required & editable on the front-end, so basically users should not create an account without filling such fields, thus they should appear in the registration form for sure. We'll possible add a separate flag later if customers ask for an option to move the optional fields to the registration form.